app/storage/metadata_parser.py

```diff
@@ -42,7 +42,7 @@ def generate(self):
 
 
 metadata_fields = {
-    "jti": MetadataField(mandatory=False),
+    "jti": MetadataField(),
```

Comment:
I think we should consider moving the validation of the `jti` claim into `token_helper.py` - it's a JWT standard claim rather than an ONS metadata claim (longer term this would then be included in the [sdc-cryptography](https://github.com/ONSdigital/sdc-cryptography) library)